### PR TITLE
Update ESmy009.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Newly revised MY manuscript.

### CODE REVIEW

File: ES/ESmy009.xml [hunk 1]
```diff
@@ -30,95 +30,1407 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                                                                                      <title type="complete" ref="LIT5282MiracleEspousing"></title>
+                                                                                      <textLang mainLang="gez"/>
+                                                                                      <incipit xml:lang="gez"><gap reason="ellipsis" resp="DN"/>
+                                                                                      ወካዕበ፡ ንነግረክሙ፡ ኦውሉደ፡ ቤተ፡ ክርስቲያን፡ ዜናሃ፡ ለእግዝእት<cb n="b"/>
+                                                                                      ነ፡<gap reason="ellipsis" resp="DN"/> ዘከመ፡
+                                                                                      መልአክ፡ ወዜነዋ፡ ትፍሥሕት፡ ወዘከመ፡ ወረደ፡ ወተሠገወ፡ እምኔሃ፨
+                                                                                      </incipit>
+                                                                                        </msItem>
+                                                                                    </msItem>
```
Comment (eu-genia) [inline ES/ESmy009.xml]: delete the second `</msItem>`

Comment (eu-genia) [reply ES/ESmy009.xml]: (and add one at the end)

File: ES/ESmy009.xml [hunk 2]
```diff
@@ -30,95 +30,1407 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                                                      <textLang mainLang="gez"/>
+                                                      <incipit xml:lang="gez"><gap reason="ellipsis" resp="DN"/>
+                                                      ወሀሎ፡ ፩ፈረሳዊ፡ ዘትለአካ፡ ለእግዝእትነ፡<gap reason="ellipsis" resp="DN"/> በኵሉ፡ ኃይሉ፡ ወይሳለማ፡
+                                                      በፍርሃት፨ ወትረ፡ ወባሕቱ፡ ኮነ፡
+                                                      ውእቱ፡ አይሁዳዊ፡ ዘይዘሩ፡ ንዋዮ፡ በከንቱ፡ ወእምዝ፡ ሐልቀ፡ ንዋዩ፨
+                                                      </incipit>
+                                                      <note>Cp. <ref target="ms_i1.3.91"/>.</note>
+                                                      </msItem>
+
+
```
Comment (eu-genia) [inline ES/ESmy009.xml]: add   `</msItem>`

File: ES/ESmy009.xml [hunk 3]
```diff
@@ -30,95 +30,1407 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <idno>MY-009</idno>
```
Comment (eu-genia) [inline ES/ESmy009.xml]: images should not be visible? if you need the viewer 
`<idno facs="MY/009/MY-009" n="115">MY-009</idno>`
(if images are uploaded)

**Review by eu-genia — CHANGES_REQUESTED** [review]
impressive

File: ES/ESmy009.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy009.xml]: ok

File: ES/ESmy009.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy009.xml]: ok

File: ES/ESmy009.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy009.xml]: I do not know if they are uploaded (possibly not), but they should be visible.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**